sdk/react/src/asyncDVCProvider.tsx

```diff
@@ -8,10 +8,8 @@ type Props = {
 }
 
 /**
- *
- * @deprecated Use the `useIsDVCInitialized` hook to block rendering of your application
+ * @deprecated Use the `useIsDevCycleInitialized` hook to block rendering of your application
  *             until SDK initialization is complete
- *
  */
 export default async function asyncWithDVCProvider(
```

Comment:
should this be changed to asyncWithDevCycleProvider?

Reply:
this method is already depricated so no.